metadata.json

```diff
@@ -1,14 +1,30 @@
 {
-  "name": "tragiccode-stackify",
+  "name": "puppet-stackify",
   "version": "0.1.0",
-  "author": "tragiccode",
+  "author": "voxpupuli",
   "summary": "Installs stackify APM agent",
   "license": "Apache-2.0",
-  "source": "https://github.com/tragiccode/tragiccode-stackify",
-  "project_page": "https://github.com/tragiccode/tragiccode-stackify",
-  "issues_url": "https://github.com/tragiccode/tragiccode-stackify/issues",
+  "source": "https://github.com/voxpupuli/puppet-stackify",
+  "project_page": "https://github.com/voxpupuli/puppet-stackify",
+  "issues_url": "https://github.com/voxpupuli/puppet-stackify/issues",
   "dependencies": [
     {"name":"puppetlabs-stdlib","version_requirement":">= 4.0.0 < 5.0.0"}
   ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "windows",
+      "operatingsystemrelease": [
+        "2012",
+        "2012R2",
+        "2016"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.0.0 < 6.0.0"
+    }
+  ],
   "data_provider": null
```

Comment:
I am not sure if data_provider is still valid? It may got deprecated.

Reply:
Removed and Done.